Cargo.toml

```
@@ -4,3 +4,5 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
+digest = "0.10.7"
+sha3 = "0.10.8"
```

Now that this is only used for testing we could make it a dev dependency only. Users that choose other hashes will not need to download and compile the one they don't use.
```suggestion

[dev-dependencies]
sha3 = "0.10.8"
```